src/modal/modal.ts

```diff
@@ -547,6 +600,45 @@ export class CaptureEyeModal extends LitElement {
     }
   }
 
+  private handleInspectContentCredentials() {
+    if (this._c2paNid) {
+      window.open(this._c2paNid);
```

Comment:
```suggestion
      window.open(this._c2paNid, '_blank', 'noopener,noreferrer');
```
Consider having these arguments for more consistent behavior and security for older browsers